src/main/java/mServer/crawler/sender/arte/ArteVideoDetailsDeserializer.java

```diff
@@ -293,7 +293,9 @@ private static String getBroadcastDateIgnoringCatchupRights(JsonArray broadcastA
                 String type = broadcastObject.get(JSON_ELEMENT_BROADCASTTYPE).getAsString();
                 
                 if(type.equals(broadcastType)) {
-                    broadcastDate = (broadcastObject.get(JSON_ELEMENT_BROADCAST).getAsString());
+                    if (!broadcastObject.get(JSON_ELEMENT_BROADCAST).isJsonNull()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)